build.gradle

```diff
@@ -50,21 +50,37 @@ subprojects {
       javadoc = true
       tests = false
     }
+    nexus {
+      def gitProvider = new nebula.plugin.info.scm.GitScmProvider()
+      def shortHash = gitProvider.calculateChange(new File('.'))
```

Comment:
The "scm provider" is something internal to the nebula info plugin. If you want the git hash, use jgit, which is what GitScmProvider uses.